modules/ROOT/pages/notifications.adoc

```diff
@@ -0,0 +1,15 @@
+= Important Notifications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+== Release Announcements
+
+New releases are announced at http://trust.mulesoft.com .
```

Comment:
under "Scheduled Maintenance" section

Reply:
and in API Community Manager release notes 
https://docs.mulesoft.com/release-notes/api-community-manager/api-community-manager-release-notes

Reply:
Thank you. Fixed.